main.go

```diff
@@ -181,6 +181,10 @@ func wrapMain() error {
 
 	secrets := map[string]interface{}{}
 	for k, v := range vargs.Secrets {
+		if v == nil {
```

Comment:
Gotcha. In that case, should I just change the struct to `map[string]string`?

https://github.com/NYTimes/drone-gke/commit/c44177f202bee4bc49290e951487b50d76ff635c#diff-7ddfb3e035b42cd70649cc33393fe32cR29